controller/apps.go

```diff
@@ -67,7 +70,13 @@ func getAndCheckOSIOClient(ctx context.Context) *OSIOClient {
 		scheme = witurl.Scheme
 	}
 
-	oc := NewOSIOClient(ctx, scheme, host)
+	// TODO: Change timeout for http client
+	httpClient := http.DefaultClient
+	wc := witclient.New(goaclient.HTTPClientDoer(httpClient))
+	wc.Host = host
+	wc.Scheme = scheme
+	wc.SetJWTSigner(goasupport.NewForwardSigner(ctx))
+	oc := NewOSIOClient(wc, &IOResponseReader{})
 
```

Comment:
I think this injection of implementation into apps.go should be avoidable; can this be put back whence it came from?